Makefile

```diff
@@ -34,7 +35,6 @@ deb: build
 	-t deb \
 	-v $(VERSION) \
 	--deb-no-default-config-files \
-	--deb-use-file-permissions \
```

Comment:
Thanks!

On Friday, August 5, 2016, Kareem Zidane <notifications@github.com> wrote:

> In Makefile <https://github.com/cs50/c/pull/19#discussion_r73718371>:
>
> > @@ -34,7 +35,6 @@ deb: build
> >  	-t deb \
> >  	-v $(VERSION) \
> >  	--deb-no-default-config-files \
> > -	--deb-use-file-permissions \
>
> OK. so after a bit more search, here's the relevant parts of a question
> and its answer
> <https://groups.google.com/forum/embed/#!topic/fpm-users/ra9ifNkX8s8> by
> Jordan Sissel (author):
>
> *question:* if I want to change the permissions of the files being
> installed with a deb file what is the best approach for this?
> *answer:* use --deb-user and --deb-group to force user/group on all files
> in the package; use chmod prior to packaging to set permissions (but
> probably ownership)
>
> fyi, per the docs, if you don't specify --deb-user and --deb-group,
> default is root and root respectively. still confused about the usage of
> --deb-use-file-permissions, so opened an issue here: jordansissel/fpm#1171
> <https://github.com/jordansissel/fpm/issues/1171>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/cs50/c/pull/19/files/d2ba20c47be3e6192ec9954ae4542140c8fe2fbd#r73718371>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAwIxl9x42gz8dsQWOQGgpmIIp3tIzC1ks5qc2CpgaJpZM4Jdvzr>
> .
>